Src/GoogleApis.Auth.DotNet4/OAuth2/GoogleWebAuthorizationBroker.cs

```diff
@@ -113,17 +113,22 @@ public class GoogleWebAuthorizationBroker
         /// <param name="taskCancellationToken">Cancellation token to cancel an operation.</param>
         /// <param name="dataStore">The data store, if not specified a file data store will be used.</param>
         /// <returns>User credential.</returns>
-        private static async Task<UserCredential> AuthorizeAsyncCore(
+        public static async Task<UserCredential> AuthorizeAsyncCore(

```

Comment:
Understood, and done. I misinterpreted your original comment, my bad, again thanks for clarifying. I removed -Core from the method name. Not that it came up but I don't think the method summary needs changing, it's still accurate since nothing there changed other than renaming and being made public.